codice/ddf-ui
# Add timezone support for timeline

- Migrates the timeline from atlas
- The timeline labels and x-axis now reflect the user's timezone.
- Changes to the user's format and timezone cause the timeline to update
- Migrates other code from atlas and removes the dependency

Note: there's a quirk with format comparisons that results in special x-axis labels to fall on times other than midnight. 

To test:
- Ingest resources
- Navigate to search and open the timeline and inspector
- Search so your items show up on the timeline
- Use the item's creation time to sanity check the timeline's values
- Check the hover label, brush labels (click and drag), and confirm that changing the user's timezone and format settings updates the timeline (note: changing the format should only change the 12 vs 24-hour display)
- Verify that dropdowns and dialogs still work as expected since that code has been migrated from atlas

![image](https://user-images.githubusercontent.com/22667297/97023134-ea9d5300-1509-11eb-9aa0-a6cfd5c11e6a.png)
![image](https://user-images.githubusercontent.com/22667297/97023227-0c96d580-150a-11eb-882b-75388b7d4cf3.png)
